github/ci/prow/files/jobs/kubevirt/kubevirt-presubmits-0.13.yaml

```diff
@@ -0,0 +1,267 @@
+periodics: null
+postsubmits: {}
+presubmits:
+  kubevirt/kubevirt:
```

Comment:
I don't think that they will work out of the box on 0.13, but we can fix it if necessary later on.